src/datasource.test.ts

```diff
@@ -1855,24 +1855,6 @@ describe('OpenSearchDatasource', function (this: any) {
         } as MetricAggregation,
       ],
     });
-
-    it('can handle multiple trace list queries', async () => {
```

Comment:
Do these tests have backend equivalents/are they just unnecessary now?

Reply:
It looks like there are some missing, added them!